docker-compose.yml

```diff
@@ -41,23 +41,23 @@ services:
       - "./logs/:/var/www/MISP/app/tmp/logs/:delegated"
       - "./files/:/var/www/MISP/app/files/:delegated"
       - "./ssl/:/etc/nginx/certs/:delegated"
+      - "./gnupg/:/var/www/MISP/.gnupg/:delegated"
       - "${PUBLIC_MOUNT_POINT}:/mnt/public/:delegated"
 #      - "./examples/custom-entrypoint.sh:/custom-entrypoint.sh" # Use the example custom-entrypoint.sh
-      - "./.gnupg/:/var/www/MISP/.gnupg/:delegated"
     environment:
       - "HOSTNAME=https://localhost"
       - "REDIS_FQDN=redis"
       - "INIT=true"             # Initialze MISP, things includes, attempting to import SQL and the Files DIR
       - "CRON_USER_ID=1"        # The MISP user ID to run cron jobs as
       # Synchronization Servers settings
-      - "SYNCSERVERS=1"
+      - "SYNCSERVERS=${SYNCSERVERS}"
       - "SYNCSERVERS_1_NAME=${SYNCSERVERS_1_NAME}"
       - "SYNCSERVERS_1_UUID=${SYNCSERVERS_1_UUID}"
       - "SYNCSERVERS_1_KEY=${SYNCSERVERS_1_KEY}"
       - |
         SYNCSERVERS_1_DATA=
         {
-          "url": "https://intel.thedfirreport.com/",
+          "url": "${SYNCSERVERS_1_URL}",
```

Comment:
ansible-environments should be updated as well to reflect these changes

Reply:
ACK